### PR TITLE
e2e shared-routes2

### PR DESCRIPTION
**Pull request by Andranik-mikhaielian**
Screenshot 2022-12-21 at 3 19 06 PM
Screenshot 2022-12-21 at 3 19 28 PM

### CODE REVIEW

File: cypress/fixtures/constants.ts [hunk 1]
```diff
@@ -213,7 +213,14 @@ export class Constants {
         sharedContextApp1H2: 'App 1',
         sharedContextApp2H2: 'App 2',
         sharedContextApp1Paragraph: 'Welcome, Billy',
-        sharedContextApp2Paragraph: 'Welcome, Susan',
+        sharedContextApp2Paragraph: 'Welcome, Susan',,
+        sharedRoutes2App1Header: 'App 1',
```
Comment (edchai) [inline cypress/fixtures/constants.ts]: we already have App1 - App2 constants, please use them

File: shared-routes2/e2e/CheckApp2.cy.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import {BaseMethods} from "../../cypress/common/base";
+import {baseSelectors} from "../../cypress/common/selectors";
+import {Constants} from "../../cypress/fixtures/constants";
+import * as constants from "constants";
+const basePage: BaseMethods = new BaseMethods()
+
+describe('Check Shared-routes App2 starts and running', () => {
```
Comment (edchai) [inline shared-routes2/e2e/CheckApp2.cy.ts]: no need for separate files. As I see the only difference in checks are hosts. so merge it one commonChecks file and walk through hosts by forEach

**Review by edchai — CHANGES_REQUESTED** [review]

File: shared-routes2/e2e/CheckApp2.cy.ts (hunk 2 above)
Comment (Andranik-mikhaielian) [reply shared-routes2/e2e/CheckApp2.cy.ts]: fixed

File: cypress/fixtures/constants.ts (hunk 1 above)
Comment (Andranik-mikhaielian) [reply cypress/fixtures/constants.ts]: fixed

**Review by edchai — APPROVED** [review]

**Pull request merged**